src/main/java/net/javadiscord/javabot2/Bot.java

```diff
@@ -60,8 +71,12 @@ public static void main(String[] args) {
 				.setToken(config.getSystems().getDiscordBotToken())
 				.setAllIntentsExcept(Intent.GUILD_MESSAGE_TYPING, Intent.GUILD_PRESENCES, Intent.GUILD_VOICE_STATES)
 				.login().join();
+
+		initListeners(api);
+
```

Comment:
Add listeners after the config has been loaded, preferably after the slash command listener has been added.